From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luai Okasha <130397131+okashaluai@users.noreply.github.com>
Date: Mon, 21 Sep 2026 19:20:47 +0200
Subject: [PATCH] 9662: Fix HTML formatting for attributes description

Fix HTML formatting for attributes description, zero or more
 is ambiguous since size attributes is required and not optional
---
 GhidraDocs/languages/html/sleigh_definitions.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GhidraDocs/languages/html/sleigh_definitions.html b/GhidraDocs/languages/html/sleigh_definitions.html
index 9ecc2ad8eb..7d83724ea1 100644
--- a/GhidraDocs/languages/html/sleigh_definitions.html
+++ b/GhidraDocs/languages/html/sleigh_definitions.html
@@ -82,7 +82,7 @@ define space <span class="bold"><strong>spacename attributes</strong></span> ;
 </pre></div>
 <p>
 The <span class="emphasis"><em>spacename</em></span> is the name of the new space,
-and <span class="emphasis"><em>attributes</em></span> looks like zero or more of the
+and <span class="emphasis"><em>attributes</em></span> looks like one or more of the
 following lines:
 </p>
 <div class="informalexample"><pre class="programlisting">
-- 
2.45.1

